app/styles/_core.less

```diff
@@ -53,6 +53,15 @@
       margin-top: 0;
     }
   }
+  .section-header{
```

Comment:
sorry nevermind about the page header, forgot storage had multiple headers, i still hate to re-do all the margins though, could we do something like

```
.section-header {
  .page-header();
  border-bottom: none;
  padding-bottom: 10px; // or 9px whatever it needs to be
}
```